api/envoy/extensions/filters/http/cache/v3alpha/cache.proto

```diff
@@ -18,8 +18,8 @@ option (udpa.annotations.file_status).work_in_progress = true;
 option (udpa.annotations.file_status).package_version_status = ACTIVE;
 
 // [#protodoc-title: HTTP Cache Filter]
-// [#extension: envoy.filters.http.cache]
 
+// [#extension: envoy.filters.http.cache]
```

Comment:
seems like thiss one was just a ws typo on the annotation

Reply:
ive had to append `.simple_http_cache` to this one - im guessing this is correct